plugins/module_utils/ldap.py

```diff
@@ -78,7 +78,7 @@ def _find_dn(self):
                 dns = self.connection.search_s(','.join(explode_dn[1:]),
                                                ldap.SCOPE_ONELEVEL, "(%s)" % explode_dn[0])
                 if len(dns) == 1:
-                    dn, attrs = dns[0]
+                    dn, _ = dns[0]
```

Comment:
```suggestion
                    dn, dummy = dns[0]
```

Reply:
Why not use the designated throw-away var?

Reply:
https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/no-underscore-variable.html